build.sbt

```diff
@@ -358,7 +358,7 @@ lazy val docs = http4sProject("docs")
       if (Http4sGhPages.buildMainSite) siteMappings.value
       else {
         val (major, minor) = apiVersion.value
-        val prefix = s"/v${major}.${minor}/"
+        val prefix = s"/v${major}.${minor}-cats/"
```

Comment:
I think the trailing slash will prevent two lines down from eating us, but that's worth checking after this lands.